examples/oneapi/dpc/source/knn/knn_search_brute_force_dense_batch.cpp

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
```

Comment:
Is there `cpp` example?

Reply:
cpp example for what?

Reply:
Please contact @KalyanovD and @PetrovKP to force it.